src/js/fluxcontroller.js

```diff
@@ -77,6 +77,11 @@ define(function (require, exports, module) {
     var MAX_RETRY_WINDOW = 6400;
 
     /**
+    *@type {Object}
```

Comment:
Broken comment formatting, plus this variable should be part of the object state. And private.